File: sandbox/php/src/TeamUpdateDefaultExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();

$team_update_request = (new Dropbox\Sign\Model\TeamUpdateRequest())
    ->setName("New Team Name");

try {
    $response = (new Dropbox\Sign\Api\TeamApi(config: $config))->teamUpdate(
        team_update_request: $team_update_request,
    );

    print_r($response);
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling Team#teamUpdate: {$e->getMessage()}";
}

File: sandbox/dotnet/src/Dropbox.SignSandbox/TeamUpdateDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class TeamUpdateDefaultExample
{
    public static void Run()
    {
        var config = new Configuration();

        var teamUpdateRequest = new TeamUpdateRequest(
            name: "New Team Name"
        );

        try
        {
            var response = new TeamApi(config).TeamUpdate(
                teamUpdateRequest: teamUpdateRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Team#TeamUpdate: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
